instrumentation/jaxrs/jaxrs-2.0/jaxrs-2.0-testing/src/main/groovy/JaxRsHttpServerTest.groovy

```diff
@@ -73,6 +104,15 @@ abstract class JaxRsHttpServerTest<S> extends HttpServerTest<S> {
     true
   }
 
+  private static boolean supportsJaxRs21() {
```

Comment:
If this is false normally and true when running latest deps, recommend being more explicit about it like our redisson instrumentation

https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/master/instrumentation/redisson-3.0/redisson-3.0.gradle#L21

Reply:
👍 
That's much better, done.